autoware_launch/config/system/system_error_monitor/diagnostic_aggregator/system.param.yaml

```diff
@@ -0,0 +1,15 @@
+/**:
+  ros__parameters:
+    system:
+      type: diagnostic_aggregator/AnalyzerGroup
+      path: system
+      analyzers:
+        debug_data_logger:
+          type: diagnostic_aggregator/AnalyzerGroup
+          path: debug_data_logger
+          analyzers:
+            storage_error:
+              type: diagnostic_aggregator/GenericAnalyzer
+              path: storage_error
+              contains: ["bagpacker"]
```

Comment:
I'll create a discussion issue for this because it's TIER IV specific.

Reply:
https://github.com/autowarefoundation/autoware_launch/issues/212

Reply:
Thank you so much.